pantheon-bundle/src/main/java/com/redhat/pantheon/extension/HydraIntegration.java

```diff
@@ -241,32 +242,26 @@ public void checkServerTrusted(X509Certificate[] certs, String authType) {
     private String buildModuleVersionUri(ModuleVersion moduleVersion) {
         StringSubstitutor strSubs = new StringSubstitutor();
         HashMap values = Maps.newHashMap();
-        values.put("moduleUuid", moduleVersion.getParent().getParent().getParent().getParent().uuid().get());
-        values.put("localeId", moduleVersion.getParent().getParent().getParent().getName());
-        values.put("variantName", moduleVersion.getParent().getName());
+        values.put("variantUuid", moduleVersion.getParent().getValueMap().containsKey(JcrConstants.JCR_UUID) ?
```

Comment:
Not a blocker, but we should add the uuid field to the `ModuleVariant` model so we can use it.